src/fmod_studio.rs

```diff
@@ -8,16 +8,35 @@ use libfmod::ffi::FMOD_STUDIO_INIT_LIVEUPDATE;
 use libfmod::ffi::{
     FMOD_INIT_3D_RIGHTHANDED, FMOD_STUDIO_INIT_NORMAL, FMOD_STUDIO_LOAD_BANK_NORMAL,
 };
-use libfmod::Studio;
+use libfmod::{Studio, System};
 
 #[derive(Resource)]
 pub struct FmodStudio(pub Studio);
 
 impl FmodStudio {
-    pub(crate) fn new(banks_paths: &[&'static str]) -> Self {
+    pub(crate) fn new(banks_paths: &[&'static str], plugin_paths: &[&'static str]) -> Self {
         let studio = Self::init_studio();
         let project_root = var("CARGO_MANIFEST_DIR").unwrap();
 
+        let studio_core = studio.get_core_system().unwrap();
+
+        plugin_paths.iter().for_each(|plugin_path| {
+            let mut path = canonicalize(Path::new(plugin_path))
+                .expect("Failed to canonicalize provided audio banks directory path.");
+
+            trace!("Canonicalized audio banks directory path: {:?}", path);
+
+            if path.is_relative() {
+                let relative_path = path;
+                path = PathBuf::new();
+                path.push(project_root.clone());
+                path.push(relative_path)
+            }
+
+            debug!("Loading FMOD plugins from: {:?}", path);
+            Self::load_plugin(studio_core, path.as_path());
+        });
```

Comment:
As this is basically a copy paste of the below code, I would like to see a common function.